civihr_default_theme/scss/components/_leave-calendar.scss

```diff
@@ -0,0 +1,122 @@
+$chr-leave-calendar-cell-height:                  35px;
+$chr-leave-calendar-cell-font-size:               10px;
+$chr-leave-calendar-item-requested-border-color:  #c7cbd0;
+$chr-leave-calendar-item-requested-border-height: 0.5em;
+$chr-leave-calendar-legend-size:                  20px;
+
+.chr_leave-calendar {
+
+  @media (min-width: $screen-lg-min) {
+    table-layout: fixed;
+  }
+}
+
```

Comment:
I do it on purpose (although can't say it's standardized code styling at this point). I leave a two lines space between the various elemens of a component, to make the code more readable and give some kind of visual grouping
```scss
.component {

}


.component__element {

}

.component__element__sub-element {

}


.component-other-element {

}